torch/nn/utils/rnn.py

```diff
@@ -310,12 +318,12 @@ def pad_packed_sequence(sequence, batch_first=False, padding_value=0.0, total_le
                              "total_length={} and max sequence length being {}"
                              .format(total_length, max_seq_length))
         max_seq_length = total_length
-    padded_output, lengths = torch._C._VariableFunctions._pad_packed_sequence(
+    padded_output, lengths = _VF._pad_packed_sequence(
```

Comment:
Is this because the JIT can't handle `torch._C._VariableFunctions`?

Reply:
Yes, we are using a wrapper for `_VariableFunctions` (which is _VF) for it to be accessible in JIT.